ember-toucan-core/src/components/form/radio-field.ts

```diff
@@ -6,20 +6,45 @@ import type { ToucanFormRadioControlComponentSignature } from './controls/radio'
 export interface ToucanFormRadioFieldComponentSignature {
   Element: HTMLInputElement;
   Args: {
+    /**
+     * Provide a string to this argument to render a hint message to help describe the control.
+     */
     hint?: string;
+
+    /**
+     * Sets the disabled attribute on the control.
+     */
     isDisabled?: boolean;
+
+    /**
+     * Provide a string to this argument to render inside of the label tag.
+     */
     label: string;
+
+    /**
+     * Sets the name attribute of the radio.
```

Comment:
Maybe a bit more description on this one?  i.e. what is the name attribute, and how is it different from the value attribute?

Reply:
Maybe just the description?  There's a bunch on that page, plus the linking/scrolling to the name section didn't seem to work...

> Name of the form control. Submitted with the form as part of a name/value pair

Reply:
Went ahead and updated both cases! Thanks!